cli/cmd/doc.go

```diff
@@ -149,11 +153,15 @@ func generateAnnotationsDocs() []annotationDoc {
 		},
 		{
 			Name:        k8s.ProxyIgnoreInboundPortsAnnotation,
-			Description: "Ports that should skip the proxy and send directly to the application",
+			Description: "Ports that should skip the proxy and send directly to the application. Comma-separated list of values, where each value can be a port number or a range `a-b`.",
+		},
+		{
+			Name:        k8s.ProxyOpaquePortsAnnotation,
+			Description: "Ports that skip the proxy's protocol detection mechanism and are proxied opaquely",
```

Comment:
Does it make sense to have the same format description here too? "Comma-separated list of...."

Reply:
Indeed :+1: